app/models/bookmark.rb

```diff
@@ -39,16 +45,6 @@ def self.auto_delete_preferences
   validate :bookmark_limit_not_reached
   validates :name, length: { maximum: 100 }
 
-  # we don't care whether the post or topic is deleted,
```

Comment:
I really am not sure why I did it this way before. Better to just keep the same deleted exclusion scope that everywhere else has by default.